systemd/manifest.json

```diff
@@ -21,7 +21,9 @@
   "is_public": false,
   "assets": {
     "monitors": {},
-    "dashboards": {},
+    "dashboards": {
+      "systemd": "assets/dashboards/systemd_overview.json"
```

Comment:
Can we name this something slightly more specific, maybe `Systemd - Overview`. Since these names are globally unique, it better to be specific. (There are some special cases for existing dashboards)

Reply:
Oh, didn't knew it should be unique. Updated.